src/Microsoft.AspNetCore.ResponseCaching/Interfaces/IResponseCachingResponseCacheabilityValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.ResponseCaching
+{
+    public interface IResponseCachingResponseCacheabilityValidator
```

Comment:
Maybe merge this with the response one. Is the logic usually coupled or is it completely separate?